openstack/ims/v2/members/GetMember.go

```diff
@@ -0,0 +1,40 @@
+package members
+
+import (
+	"time"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+)
+
+type GetMemberOpts struct {
+	ImageId  string `json:"-" required:"true"`
+	MemberId string `json:"-" required:"true"`
+}
+
+func GetMember(client *golangsdk.ServiceClient, opts GetMemberOpts) (*Member, error) {
```

Comment:
could you check if it works in otc?

Reply:
it is just the same as the old `imageservice` (from OpenStack).
Test in the TF provider. (I don't have another ProjectId)
will try.